### PR TITLE
LPS-156054 baseline

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-156054

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/system/SystemObjectDefinitionMetadataServicesTrackerImpl.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	service = SystemObjectDefinitionMetadataServicesTracker.class
+)
+public class SystemObjectDefinitionMetadataServicesTrackerImpl
+	implements SystemObjectDefinitionMetadataServicesTracker {
+
+	@Override
+	public SystemObjectDefinitionMetadata getSystemObjectDefinitionMetadata(
+		String key) {
+
+		return _serviceTrackerMap.getService(key);
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/system/SystemObjectDefinitionMetadataServicesTrackerImpl.java:41]: @marcelabc can you rename this for

File: modules/apps/object/object-api/src/main/java/com/liferay/object/system/SystemObjectDefinitionMetadataServicesTracker.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.system;
+
+/**
+ * @author Marcela Cunha
+ */
+public interface SystemObjectDefinitionMetadataServicesTracker {
```
Comment (gabrielwas) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/system/SystemObjectDefinitionMetadataServicesTracker.java:20]: @marcelabc can you use `SystemObjectDefinitionMetadataTrackerImpl` for this? Thx.

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/system/CPDefinitionSystemObjectDefinitionMetadata.java [hunk 3]
```diff
@@ -54,6 +54,11 @@ public Class<?> getModelClass() {
 		return CPDefinition.class;
 	}
 
+	@Override
+	public String getModelClassIdName() {
```
Comment (gabrielwas) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/system/CPDefinitionSystemObjectDefinitionMetadata.java:58]: @marcelabc can you rename it to `getIdPropertyName` ?

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:forward

**Comment by gabrielwas** [comment]
LGTM 😄

**Review by gabrielwas — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**